docs/user_guide/index.rst

```diff
@@ -9,6 +9,6 @@ User Guide
 
    fairness_in_machine_learning
    assessment
-   mitigation
+   mitigation/index
```

Comment:
Out of curiosity: will this wreak havoc with multi-version?

Reply:
It will mean that users can't just click between versions, since the files will have moved.

Reply:
as long as we have _a_ copy of the docs for each version, I don't think we should worry about how users can find a file in different versions.